README.md

```diff
@@ -216,93 +217,367 @@ By simplicity, we have three types of commits:
-    ```bash
-    git checkout main
-    git checkout -b feature/nice-new-thing
-    ```
 
+  ```bash
+  git checkout main
+  git checkout -b feature/nice-new-thing
+  ```
 - **Step 2.** Develop the contribution in your *feature branch* by making commits (see [How to make a commit](#how-to-make-a-commit) section).
 
-    ```bash
-    git add <filenames>
-    git commit -m "feat: add nice new thing"
-    ```
-
+  ```bash
+  git add <filenames>
+  git commit -m "feat: add nice new thing"
+  ```
 - **Step 3.** Push your *feature branch* to the remote repository (in the following example represented by the *origin* alias)
 
-    ```bash
-    git push origin feature/nice-new-thing
-    ```
-
+  ```bash
+  git push origin feature/nice-new-thing
+  ```
 - **Step 4.** Open a Pull Request (PR), select its reviewers and add it one of the release labels:
 
-    Release Labels | Description | Release Type
-    ---------------|-------------|-------------
-    `release-no` | When no new version should be released when the PR is merged into the `main` branch | None
-    `release-auto` | When the new version to be released should be deducted automatically based on the PR semantic commits when it is merged | [PATCH, MINOR, MAJOR]
-    `release-patch` | When the new version should be released as a patch | PATCH
-    `release-minor` | When the new version should be released as a minor | MINOR
-    `release-major` | When the new version should be released as a major | MAJOR
+  | Release Labels    | Description                                                                                                             | Release Type          |
+  | ----------------- | ----------------------------------------------------------------------------------------------------------------------- | --------------------- |
+  | `release-no`    | When no new version should be released when the PR is merged into the `main` branch                                   | None                  |
+  | `release-auto`  | When the new version to be released should be deducted automatically based on the PR semantic commits when it is merged | [PATCH, MINOR, MAJOR] |
+  | `release-patch` | When the new version should be released as a patch                                                                      | PATCH                 |
+  | `release-minor` | When the new version should be released as a minor                                                                      | MINOR                 |
+  | `release-major` | When the new version should be released as a major                                                                      | MAJOR                 |
 
-    >**Important note:** If none of the labels are added, a version release corresponding to `release-auto` will be triggered.
 
+  > **Important note:** If none of the labels are added, a version release corresponding to `release-auto` will be triggered.
```

Comment:
Did you test this? I believe it is incorrect, I think the PR is just blocked.

Reply:
Perfect! A label is required. I will remove this note.